app/views/handlers/thank_you.py

```diff
@@ -0,0 +1,51 @@
+from flask import session as cookie_session
+from werkzeug.exceptions import NotFound
+from app.views.contexts.thank_you_context import (
+    build_default_thank_you_context,
+    build_census_thank_you_context,
+)
+from app.globals import get_session_store
+
+DEFAULT_THANK_YOU_TEMPLATE = "thank-you"
+CENSUS_THANK_YOU_TEMPLATE = "census-thank-you"
+
+CENSUS_TYPE_MAPPINGS = {
+    "household": "HH",
+    "communal_establishment": "CE",
+    "individual": "IR",
+}
```

Comment:
Any reason you chose to make these module level variables instead of class attributes?